example/edb.yml

```diff
@@ -7,6 +7,10 @@
     :database: sample_database
     :host: localhost
     :port: 5432
+    :only_tables: ['table1', 'table2']
```

Comment:
I suggest to give this option an empty array to be coherent with `without_tables`.